modules/ROOT/pages/monitoring/metric-manager.adoc

```diff
@@ -2,11 +2,37 @@
 :description: This section describes the Metric manager of Neo4j Ops Manager.
 
 The metrics page offers rich charts and tables with data related to your monitored DBMSs.
-Currently three time windows are supported Live (30 minutes), 24 hours and 3 days worth of data.
+Currently six time windows are supported, from 30 minutes (live) up to 30 days.
```

Comment:
Do we specifically need to mention how many time windows we have?

Reply:
No we don't. I just updated the count :). Do you know if the shortest time window is different from the others (because it is called "live"? Does (only) "live" updates automatically?

Reply:
No, there is no difference. It just used to be called live for some reason.. Better to remove this reference.